en/docs/choreo-samples/choreo-sample-collection.md

```diff
@@ -0,0 +1,33 @@
+# Choreo Sample Collection
+
+Choreo provides a diverse collection of sample applications covering various use cases. These samples help you get started quickly by deploying applications with ease.
+
+## Prerequisites
+
+- Sign in to Choreo.  
+- If you're signing in for the first time, create an organization:  
+    1. Go to [Choreo Console](https://console.choreo.dev/) and sign in using Google, GitHub, or Microsoft.  
+    2. Enter a unique organization name, for example, `Stark Industries`.  
+    3. Accept the privacy policy and terms of use.  
+    4. Click **Create** to set up the organization.  
+- Create a project or navigate to an existing project:  
+    - If you don't have a project:  
+        1. On the organization home page, click **\+ Create Project**.  
+        2. Enter a display name, unique name, and description for the project.  
+        3. Click **Create**.  
+    - If you already have a project, go to the project overview page of your choice.
+
+## Get Started with Samples
+
+1. Scroll down to the **Get Started with a Sample** section.  
+2. You will see a few sample applications here. To access the full list of samples, click **View All Samples**.
```

Comment:
Seems like the samples section goes as  **Get Started with a Sample** when we have not created any component in a project but if we do happen to have a component it goes as **Create from a sample**. Shall we go with **Get Started with a Sample** in both instances?
I don't see the **View All Samples** option in the UI at the moment either. 
<img width="1675" alt="Screenshot 2024-10-28 at 08 54 34" src="https://github.com/user-attachments/assets/1f5df9b3-ce6b-487d-b610-2ac50b42c9da">
Are we going to add a **View All Samples** option?

Just want a confirmation so that I can document the steps properly.

Reply:
+1 for **Get Started with a Sample**

Right now the **View All Samples** button is disabled when the org component limit is reached.

However, the user reaches the sample list page by manually going to the path `organizations/{org-name}/projects/{project-id}/components/new/samples`.

AFAIU there is no harm in letting the user see the sample list even if the user cannot create or quick deploy.

@ThusharaSampath can you share your thoughts on this?

Reply:
+1 to this idea. Then the user can view samples.